src/data/equiv/basic.lean

```diff
@@ -2285,6 +2290,12 @@ protected def congr {ra : setoid α} {rb : setoid β} (e : α ≃ β)
   quotient ra ≃ quotient rb :=
 quot.congr e eq
 
+@[simp]
+lemma congr_mk {ra : setoid α} {rb : setoid β} (e : α ≃ β)
+  (eq : ∀ (a₁ a₂ : α), setoid.r a₁ a₂ ↔ setoid.r (e a₁) (e a₂)) (a : α):
+  quotient.congr e eq (quotient.mk a) = quotient.mk (e a) :=
+@quot.congr_mk _ _ (setoid.r) (setoid.r) e eq a
```

Comment:
```suggestion
rfl
```

It's actually quite important to prove things with `rfl` that can be proved with `rfl`, otherwise `dsimp` does not know about them.

Reply:
Oh fair enough. Thanks a lot for the advice !